gen3-cli/cot/backend/generate/product/base/__init__.py

```diff
@@ -0,0 +1,24 @@
+import os
+from cookiecutter.main import cookiecutter
+from cot import conf
+from cot.backend.generate import utils
+
+
+TEMPLATES_DIR = os.path.join(conf.COOKIECUTTER_TEMPLATES_BASE_DIR, 'products')
+
+
+def run(**kwargs):
+    utils.replace_parameters_values(
```

Comment:
This could be some another procedure in utils, given that these 8 lines are duplicated in many files

Reply:
Agree, but I was focused on minimization of the code in other places,  namely in the commands.  I'll push another commit which will generalize this section too.